lib/index.js

```diff
@@ -334,6 +334,20 @@ Mixpanel.prototype._parseResponse = function(fn){
   return this.handle(function(err, res){
     if (err) return fn(err);
     if (!res.body.status) return fn(self.error(res.body.error));
```

Comment:
@anoonan do we need to change this line and the one above here? Aren't these two `if` statements the ones that are exposing the errors without statuscodes?